src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefConditionalOperatorTests.cs

```diff
@@ -1263,7 +1263,7 @@ .maxstack  1
             void verify(CSharpCompilationOptions options, string expectedIL)
             {
                 var comp = CreateCompilation(code, options: options);
-                var verifier = CompileAndVerify(comp, expectedOutput: "run");
+                var verifier = CompileAndVerify(comp, expectedOutput: "run", verify: Verification.Fails);
```

Comment:
I think we typically use `Verification.Skipped` when valid code is not handled by the verifier. `Verification.Fails` seems to imply a bug.

Reply:
Other ref local code used `Fails`, which is why I defaulted to that here.